src/System.Text.Encoding/tests/ASCIIEncoding/ASCIIEncodingGetMaxByteCount.cs

```diff
@@ -2,27 +2,19 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System.Collections.Generic;
 using Xunit;
 
 namespace System.Text.Tests
 {
     public class ASCIIEncodingGetMaxByteCount
     {
-        private static readonly RandomDataGenerator s_randomDataGenerator = new RandomDataGenerator();
```

Comment:
> besides, I don't really see the random data adding any value

testing against fixed data is good but you never know what cases we are missing or if there is any corner cases has any problem. having random data in such tests can get lucky enough to hit cases you never thought about it or having it covered in your test. as I said if using random data not causing real test stability issue, then I am seeing it is good to keep. 

@stephentoub could you explain why we need to get rid of the random data?